src/solver/conv/conv_ocl_dir2D_bwdWrW_2.cpp

```diff
@@ -36,6 +36,8 @@
 
 #include <algorithm>
 
+#define WORKAROUND_MIOPEN_1392 (HIP_PACKAGE_VERSION_FLAT >= 6004000000)
```

Comment:
```suggestion
// Disable this solver due to random GPU memory access faults on gfx11 and gfx12
#define WORKAROUND_LWPMIOPEN_1392 (HIP_PACKAGE_VERSION_FLAT >= 6004000000)
```

Could you please consider using the following format for the workaround macro name:

* `WORKAROUND_MIOPEN_ISSUE_xxxx` for github issues
* `WORKAROUND_LWPMIOPEN_xxxx` for LWPMIOPEN tickets
* `WORKAROUND_SWDEV_xxxx` for SWDEV tickets

and providing a comment before it.

Reply:
Sure